### DISCUSSION TITLE
Maximum distance in the field

### DISCUSSION THREAD

**Discussion by garudaonekh** [timmbogner/Farm-Data-Relay-System#4] [General]
hi,
How far can espnow reach in case of internal antenna and as well as external antenna? I am currently using Tasmota with trouble with range in the open field.

I mainly use esp32 wroom 32U and esp8266 d1

**Comment by garudaonekh** [comment]
By the way, have you tried LR mode(long range mode) of esp32? I tried it but the range is dissapointing, i can Only reach 60m

**Comment by timmbogner** [comment]
I haven't done many range tests yet, but an early version of the system used to span about 120 meters of open field pretty easily.
I haven't tested LR mode very thoroughly. Since it isn't compatible with 8266 I've been hesitant to bother.

**Comment by garudaonekh** [comment]
120m is impressive. what kind of antenna used?and which board?

**Comment by timmbogner** [comment]
Believe it or not, it was just an ESP-01. I currently use 8266 WeMos modules generally and an ESP32 DevKit for the terminal module.

**Comment by garudaonekh** [comment]
Let me try it using ESP32-WROOM 32U(External antenna) and see how far it can reach.
I plan to use this module for the sensor(https://github.com/LilyGO/LILYGO-T-OI)
and ESP32-WROOM 32U as gateway

**Comment by timmbogner** [comment]
https://youtu.be/oz0a7Ur7nko
Here's a good video with range tests.
